academy/2-cosmos-concepts/11-testing.md

```diff
@@ -51,52 +50,64 @@ Unit tests are your classic "smallest tests possible", and focus on a single mod
+As an example, imagine that your module moves tokens on behalf of your users. Your module, therefore, has a dependency on the bank keeper. As part of a unit test setup, you create a mocked bank keeper and use it. After the test action, at verification time, your unit test confirms that your module called the expected functions of the bank keeper within the expected parameters. Your unit test does **not** test whether bank balances have changed because, remember, your module does not have a real bank keeper.
 
 ### Integration tests
 
-Integration tests are one step wider in scope. They are still focused on your module but, instead of mocking the dependencies, now your test provisions a **minimally-viable application** that includes fully-fledged dependencies, including – crucially – those your own module needs.
+Integration tests are one step wider in scope. They are still focused on your module but, instead of mocking the dependencies, now your test provisions a **minimal-viable application** that includes fully-fledged dependencies, including – crucially – those your own module needs.
 
 <HighlightBox type="info">
 
-A minimally-viable application is one that contains your module and all its dependencies, and their own continuing until all dependencies are satisfied, but nothing more.
+A minimal-viable application contains your module and all its dependencies, as well as their dependencies, but nothing more.
 
 </HighlightBox>
 
-In a well-designed testing environment, providing such fully-fledged dependencies should not be a concern of your module's tests. All the more so if your dependencies have dependencies of their own. You want to minimize such deep correlations between modules, even with regard to their tests.
+In a well-designed testing environment, providing such fully-fledged dependencies should not be a concern of your module's tests. All the more so if your dependencies have dependencies of their own. You want to minimize such deep correlations between modules, even in regard to their tests.
 
-This is why, with a view to minimizing correlations, from version 0.47 of the Cosmos SDK each module exposes functions to provision a minimally viable module. This way, your module only knows how to instantiate itself given fully-fledged dependencies, the _inputs_. An added benefit is that your module exposes explicitly the inputs it needs to instantiate.
+This is why, to minimize correlations, from version 0.47 of the Cosmos SDK on each module exposes functions to provide a minimal viable module. This way, your module only knows how to instantiate itself given fully-fledged dependencies, the _inputs_. An added benefit is that your module exposes explicitly the inputs it needs to instantiate.
```

Comment:
> ...on each module exposes functions...
Does not sound right to me.

```suggestion
This is why, to minimize correlations, from version 0.47 of the Cosmos SDK on each module exposes functions to provide a minimum viable module. This way, your module only knows how to instantiate itself given fully-fledged dependencies, the _inputs_. An added benefit is that your module exposes explicitly the inputs it needs to instantiate.
```

Reply:
Could change to "...from version 0.47 of the Cosmos SDK onwrd each module exposes..."

OR, remove "on" completely.

Reply:
aah on was for onward

Reply:
Changed. Thank you Xavier and Andrew! :)